### PR TITLE
EVA-2536 - Gitlab automatic deployment pipeline

### PR DESCRIPTION
**Pull request by tcezard**

### CODE REVIEW

**Review by sundarvenkata-EBI — APPROVED** [review]

**Review by apriltuesday — APPROVED** [review]

**Comment by apriltuesday** [comment]
Wait actually... should there be something executable here?

File: covid19dp_submission/VERSION [hunk 1]
```diff
@@ -1 +1 @@
-0.1.dev1
\ No newline at end of file
+0.1.1
```
Comment (sundarvenkata-EBI) [inline covid19dp_submission/VERSION:1]: Shall we do version bump as a separate commit to the main branch (similar to our [deployment process](https://www.ebi.ac.uk/seqdb/confluence/display/VAR/Release+and+deployment+-+SOP) for Java projects)?

**Review by apriltuesday — APPROVED** [review]
